ingestion/src/metadata/ingestion/source/pipeline/databrickspipeline/metadata.py

```diff
@@ -86,25 +91,38 @@ def create(
             )
         return cls(config, metadata)
 
-    def get_pipelines_list(self) -> Iterable[dict]:
-        for workflow in self.client.list_jobs() or []:
-            yield workflow
+    def get_pipelines_list(self) -> Iterable[DataBrickPipelineDetails]:
```

Comment:
this can return none.
```suggestion
    def get_pipelines_list(self) -> Optional[Iterable[DataBrickPipelineDetails]]:
```